### PR TITLE
[Feature] scroll down to most recent message

### PR DESCRIPTION
**Pull request by matheuspb**
# Description

https://www.pivotaltracker.com/story/show/182049543

# How to test

Go to a chat and scroll up a bit, you should see the button at the bottom left. If you touch the button, it scrolls all the way down to the most recent message.

Screen Shot 2022-05-10 at 17 28 44

<sub> Mention [stepsize] in a comment if you'd like to report some technical debt. See examples [here](https://app.stepsize.com/api/demo-pr-redirect).</sub>

### CODE REVIEW

File: src/screens/Chat/ChatView.tsx [hunk 1]
```diff
@@ -248,6 +273,20 @@ const styles = StyleSheet.create({
...
+    position: "absolute",
+    right: 8,
+    bottom: FLATLIST_BOTTOM_OFFSET + 8,
+    zIndex: 100,
+  },
+  scrollDownButton: {
+    backgroundColor: "#046DE8",
+    height: 40,
+    width: 40,
+    borderRadius: 20,
```
Comment (tgfassina) [inline src/screens/Chat/ChatView.tsx:286]: There's just a white tiny border missing here

File: src/screens/Chat/ChatView.tsx [hunk 2]
```diff
@@ -169,6 +179,21 @@ const ChatView: React.FC<Props> = ({ route, navigation }) => {
         }}
         keyExtractor={(item) => item.id}
       />
+      {showScrollDown && (
+        <View style={styles.scrollDownButtonContainer}>
+          <TouchableOpacity
+            style={styles.scrollDownButton}
+            onPress={() => {
```
Comment (tgfassina) [inline src/screens/Chat/ChatView.tsx:186]: Do you think it makes sense to take the opportunity and also call this function when pushing a message? (other apps scroll to bottom when you send something)

**Review by tgfassina — APPROVED** [review]
left a few comments, but it looks good to me. just works ✌

File: src/screens/Chat/ChatView.tsx (hunk 2 above)
Comment (matheuspb) [reply src/screens/Chat/ChatView.tsx:186]: yeah, makes sense to me, I'll do it

**Pull request merged**